Client/src/Components/UserReview/DatasetModal.tsx

```diff
@@ -0,0 +1,85 @@
+import { Dialog, DialogActions, DialogContent, DialogTitle, Grid, Typography } from '@material-ui/core'
+import { FormikProps, FormikValues } from 'formik'
+import React, { useRef, useState } from 'react'
+
+import CancelIcon from "@material-ui/icons/Cancel"
+import { DatasetForm } from '../DatasetUpload/DatasetForm'
+import { DefaultFormFooter } from '../Forms/DefaultFormFooter'
+import { IApprovedDatasetModel } from '../../Models/Datasets/IApprovedDatasetModel'
+import { IDatasetModel } from '../../Models/Datasets/IDatasetModel'
+import { ModalFooter } from './ModalFooter'
+
+interface IProps {
+  open: boolean
+  dataset: IApprovedDatasetModel,
+  setOpen: (value: boolean) => void
+  handleApproveDataset: (datasetId: number) => void,
+  handleDeleteDataset: () => void,
+  handleSubmitDataset: (formDataset: IDatasetModel) => void,
+}
+
+export const DatasetModal = (props: IProps) => {
+  const { open, dataset, setOpen, handleApproveDataset, handleDeleteDataset, handleSubmitDataset } = { ...props }
+  const [isEditMode, setEditMode] = useState(false)
+  const formikReference = useRef<FormikProps<unknown>>()
+
+  const renderHeader = () => {
+    return (
+      <DialogTitle id="confirmation-dialog-title">
```

Comment:
I accidentally copy-pasted the id, and the id doesnt describe the element its on properly, can you remove it pls